apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-common/src/main/java/co/elastic/apm/agent/esrestclient/ElasticsearchRestClientInstrumentation.java

```diff
@@ -35,4 +39,10 @@ public Collection<String> getInstrumentationGroupNames() {
         return Collections.singleton("elasticsearch-restclient");
     }
 
+    @Override
+    public ElementMatcher.Junction<ClassLoader> getClassLoaderMatcher() {
+        // ensure only ES client versions without native instrumentation are instrumented
```

Comment:
[minor] adding the ES client version number in the comment here might be relevant for future reference, alternatively we could have also used the `co.elastic.apm.agent.sdk.bytebuddy.CustomElementMatchers#implementationVersionLte` method that makes it self-documented, however doing so here would have required to already have the new client released, which is not the case yet.

Reply:
I've added the cutoff version to the comment